gui/mozregui/report.py

```diff
@@ -73,22 +82,53 @@ def clear(self):
 
     @Slot(object)
     def attach_bisector(self, bisector):
-        slots = ('step_started', 'step_build_found', 'step_testing',
-                 'step_finished', 'started', 'finished')
+        bisector_slots = ['step_started',
```

Comment:
Maybe mainly a matter of taste, but I prefer to use tuple instead of list when the data inside won't be changed, because tuple are immutable and they are here for that reason :) Same thing for downloader_slots.